assets/css/settings.scss

```diff
@@ -299,6 +299,21 @@
 			}
 		}
 	}
+	.sensei-learners-extra {
+		.postbox {
+			padding: 1em 0px 0px 1em;
+			h2 {
+				padding: 0px !important;
```

Comment:
We should try to avoid `!important` at all costs, since it means that nobody will be able to override our styles. Given third party developers build on top of Sensei, it's important that we don't make things harder for them.

Reply:
You're absolutely right, changing it to be more specific instead of using important